ignite/templates/ibc/packet.go

```diff
@@ -171,76 +174,84 @@ func moduleModify(replacer placeholder.Replacer, opts *PacketOptions) genny.RunF
 	}
 }
 
-func protoModify(replacer placeholder.Replacer, opts *PacketOptions) genny.RunFn {
+// Modifies packet.proto to add a field on the oneof element of the message created and
+// add a couple of messages.
+//
+// What it depends on:
+//   - Existence of a Oneof field named 'packet'.
+func protoModify(opts *PacketOptions) genny.RunFn {
 	return func(r *genny.Runner) error {
 		path := filepath.Join(opts.AppPath, "proto", opts.AppName, opts.ModuleName, "packet.proto")
 		f, err := r.Disk.Find(path)
 		if err != nil {
 			return err
 		}
-
-		content := f.String()
-
-		// Add the field in the module packet
-		fieldCount := strings.Count(content, PlaceholderIBCPacketProtoFieldNumber)
-		templateField := `%[1]v
-				%[2]vPacketData %[3]vPacket = %[4]v; %[5]v`
-		replacementField := fmt.Sprintf(
-			templateField,
-			PlaceholderIBCPacketProtoField,
-			opts.PacketName.UpperCamel,
-			opts.PacketName.LowerCamel,
-			fieldCount+2,
-			PlaceholderIBCPacketProtoFieldNumber,
```

Comment:
This constant and probably a couple of others can probably be removed from `ignite/templates/ibc/placeholders.go`

Reply:
Yup, I was planning to remove all placeholder in the final final PR just to keep it all localized in a single PR. Lmk if I should just drop them as I go.

Reply:
OK for me.